src/Storage/Storage.Management.Test/Storage.Management.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="9.1.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="10.0.0" />
```

Comment:
Please release nuget package - this cannot be merged until that is done.

Reply:
Will change it to released SDK.

Reply:
Updated, by remove the private package in LocalFeed.
Since the private and released one has same version, project file don't need change.